apt.txt

```diff
@@ -17,6 +17,7 @@ libgsl0-dev
 libi2c-dev
 libjpeg-dev
 liblua5.2-dev
+libpcl-dev
```

Comment:
Hi @traversaro,
Does installing `libpcl-dev` ensure that `vtk` gets also installed?

Reply:
Not sure, i still need to check

Reply:
My hunch is that, even in case `vtk` is correctly installed under the hood, it'd be more convenient to list it down explicitly, given it's quite a prominent package.

Reply:
The problem is that depending on the ubuntu distribution there is a different package, i.e. `libvtk8-dev`, `libvtk9-dev`, `libvtk10-dev`. `libpcl-dev` is always the same and consistently installs also the correct `libvtk*-dev`, so it seems more mantainable to just document to install `libpcl-dev`.

Reply:
Clear!